src/main/webapp/auction/create.jsp

```diff
@@ -13,12 +13,13 @@
 </head>
 <body>
 <div class="container-fluid form-container p-4">
-    <form method="post">
+    <form method="post" enctype="multipart/form-data">
         <%
             var title = request.getAttribute("title") == null ? "" : request.getAttribute("title");
             var description = request.getAttribute("description") == null ? "" : request.getAttribute("description");
             var minBid = request.getAttribute("minBid") == null ? "" : request.getAttribute("minBid");
             var endDate = request.getAttribute("endDate") == null ? "" : request.getAttribute("endDate");
+            var auctionImage = request.getAttribute("auctionImage") == null ? "" : request.getAttribute("auctionImage");
```

Comment:
Is this attribute being set anywhere?

You can test it with this scenario:
* Add an image
* Fill in other fields
* Intentionally leave a validation error in one of the other fields (eg: incorrect date format for end date)
* Submit the form

The create auction page will reload with a list of errors and all the fields will be filled with whatever values were submitted last time. In this page, the auction image uploaded in the previous step should also be displayed somewhere here.. i guess we might have to add an `<img>` tag or something and render the image as base64. or maybe there's a better way. I'll leave that up to you